src/components/Row.js

```diff
@@ -1,7 +1,7 @@
 import React, { PropTypes } from 'react';
 import classNames from 'classnames';
 import createProps from '../createProps';
-import style from 'flexboxgrid';
+import style from 'style!css?modules!flexboxgrid';
```

Comment:
In [README.md](https://github.com/roylee0704/react-flexbox-grid#basic-webpack-configuration) we give instructions how to set up webpack loaders for `flexboxgrid` to be parsed using CSS Modules, so I suggest leaving this as `import style from 'flexboxgrid';`, because in the future we might make CSS Modules optional (see #60), so I'd prefer not having webpack-specific code like this.